package.json

```
@@ -3,27 +3,40 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3001",
     "build": "next build",
     "start": "next start",
```

```suggestion
    "start": "next start -p 3001",
```
recommended for consistency in startup